rand_core/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+- Derive PartialEq+Eq for BlockRng and BlockRng64
+- Add PartialEq+Eq constraint to BlockRngCore::Results
```

Comment:
Is this reasonable?  Also should `BlockRngCore` itself have a `where Self: PartialEq + Eq` constraint at this point?